pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -142,136 +184,201 @@ type KubeletConfiguration struct {
 	Authorization KubeletAuthorization `json:"authorization"`
 	// registryPullQPS is the limit of registry pulls per second.
 	// Set to 0 for no limit.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact scalability by changing the amount of traffic produced
+	// by image pulls.
 	// Default: 5
 	// +optional
 	RegistryPullQPS *int32 `json:"registryPullQPS,omitempty"`
 	// registryBurst is the maximum size of bursty pulls, temporarily allows
 	// pulls to burst to this number, while still not exceeding registryPullQPS.
 	// Only used if registryPullQPS > 0.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact scalability by changing the amount of traffic produced
+	// by image pulls.
 	// Default: 10
 	// +optional
 	RegistryBurst int32 `json:"registryBurst,omitempty"`
 	// eventRecordQPS is the maximum event creations per second. If 0, there
 	// is no limit enforced.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact scalability by changing the amount of traffic produced by
+	// event creations.
 	// Default: 5
 	// +optional
 	EventRecordQPS *int32 `json:"eventRecordQPS,omitempty"`
 	// eventBurst is the maximum size of a burst of event creations, temporarily
 	// allows event creations to burst to this number, while still not exceeding
 	// eventRecordQPS. Only used if eventRecordQPS > 0.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact scalability by changing the amount of traffic produced by
+	// event creations.
 	// Default: 10
 	// +optional
 	EventBurst int32 `json:"eventBurst,omitempty"`
 	// enableDebuggingHandlers enables server endpoints for log collection
 	// and local running of containers and commands
```

Comment:
nit: May want to mention that features such as exec, portforward, logs, attach will not be supported if the field is false.

Reply:
Done, thanks.